src/config/directory_tweak.cc

```diff
@@ -100,7 +100,7 @@ std::shared_ptr<DirectoryTweak> DirectoryConfigList::get(const fs::path& locatio
 {
     AutoLock lock(mutex);
     for(auto testLoc = location.has_filename() ? location.parent_path() : location; testLoc.has_parent_path() && testLoc != "/"; testLoc = testLoc.parent_path()) {
-        auto entry = std::find_if(list.begin(), list.end(), [=] (const auto& d) { return ((d->getLocation() == location || d->getInherit() == true) && d->getLocation() == testLoc); }); 
+        auto entry = std::find_if(list.begin(), list.end(), [=](const auto& d) { return ((d->getLocation() == location || bool(d->getInherit())) && d->getLocation() == testLoc); });
```

Comment:
no need for a cast getInherit() returns a bool

Reply:
Oddly enough this change was automated...